src/EditorFeatures/TestUtilities/Remote/WatsonReporter.cs

```diff
@@ -0,0 +1,23 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+ 
+ using System;
+ 
+ namespace Microsoft.VisualStudio.LanguageServices.Implementation
+ {
+     /// <summary>
+     /// Mock to make test project build
+     /// </summary>
+     internal class WatsonReporter
+     {
+         public static void Report(string description, Exception exception, Func<IFaultUtility, int> callback)
```

Comment:
this one is mock. it won't do anything. the JsonRpcClient file is linked in the test project which doesnt have WatsonReport. so the mock is created there.

in production code, it will use http://source.roslyn.io/#Microsoft.VisualStudio.LanguageServices/Implementation/Watson/WatsonReporter.cs,48